interface/frontend/src/views/OpenImage.vue

```diff
@@ -1,13 +1,129 @@
 <template>
-  <image-series></image-series>
+  <div class="container">
+    <div class="col-xs-6">
+      <div style="width:512px;height:512px;position:relative;display:inline-block;color:white;"
+        oncontextmenu="return false"
+        class='cornerstone-enabled-image'
+        unselectable='on'
+        onselectstart='return false;'
+        onmousedown='return false;'>
+
+        <div ref="DICOM" style="width:512px;height:512px;top:0px;left:0px; position:absolute;"></div>
+      </div>
+      <p>{{ dicom.imageId}}</p>>
+    </div>
+  </div>
 </template>
 
 <script>
 import ImageSeries from '../components/ImageSeries'
+var cornerstone = require('cornerstone-core')
+var jquery = require('jquery')
 
 export default {
   components: {
     ImageSeries
+  },
+  name: 'dicom-view',
+  data () {
+    return {
+      dicom: {
+        imageId: '',
+        minPixelValue: 0,
+        maxPixelValue: 255,
+        slope: 1.0,
+        intercept: 0,
+        windowCenter: 127,
+        windowWidth: 70,
+        render: cornerstone.renderGrayscaleImage,
+        getPixelData: this.getPixelData,
+        rows: 512,
+        columns: 512,
+        height: 512,
+        width: 512,
+        color: false,
+        base64data: '',
+        columnPixelSpacing: 1,
+        rowPixelSpacing: 1,
+        sizeInBytes: 512 * 512 * 2
+      },
+      dicomUrl: 'LIDC://LIDC-IDRI-0002/1.3.6.1.4.1.14519.5.2.1.6279.6001.490157381160200744295382098329/1.3.6.1.4.1.14519.5.2.1.6279.6001.619372068417051974713149104919/-80.750000.dcm',
+      info: null,
+      csName: 'LIDC'
+    }
+  },
+  watch: {
+    info: function (val) {
+      if (val != null) {
+        this.applyMeta(val)
+        console.log(this.dicom)
+        this.loadImage(this.resolveDICOM)
+      }
+    }
+  },
+  mounted: function () {
+    this.fetchData(this.dicomUrl)
+  },
+  methods: {
+    fetchData (id) {
+      this.$http.get('/api/images/metadata?dicom_location=/images/' + id.slice(this.csName.length + 3))
```

Comment:
To align with the other component, it's best to use the axios API instead.

Reply:
Implemented [here](https://github.com/concept-to-clinic/concept-to-clinic/pull/188/commits/fa9eee3ff7707202cbdafd60d263e1661cd1b1cf#diff-70d179422138a3042c2135a282065626R59)